### DISCUSSION TITLE
Medicated Cream for Androids

### DISCUSSION THREAD

**Discussion by bwbfat** [hydrogen-music/hydrogen#1799] [Ideas]
I really would love hydrogen on Android and can't imagine I'm not the only one. I'm also a software dev and wonder if I came up with a really functional work flow, if anyone has the experience with code for hydrogen if they'd be interested to work on this.

I bet hydrogen is too great for the phone, but what I'm really after is the pattern base midi-like that can at least tone change quality samples. Doable? Phones can be really comfortable, and for art that's good. Gmail bullfrogma and thanks for reading this. Nooo. I didn't do a search for similar discussions. It took me too long to post this, I can't turn back now. Clicks button. Feels terrible.

**Comment by bwbfat** [comment]
Long story short Id love to make chiptunes with densely analog samples. On the phone would be amazing. If touch screen work flow is a problem I think there's plenty of solutions.

**Comment by bwbfat** [comment]
I still feel terrible.

**Comment by theGreatWhiteShark** [comment]
Answered on mailing list (see https://sourceforge.net/p/hydrogen/mailman/hydrogen-devel/thread/CA%2B%3DPdRU-o1gt80k3yjwd8XApwQgriFasURCT%3D35-4jx%2BD0BE4g%40mail.gmail.com/#msg37859830)
